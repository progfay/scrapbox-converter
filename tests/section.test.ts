/* global describe it expect */

import { parse } from '@progfay/scrapbox-parser'
import convert from '../src'
import pretty from 'pretty'

describe('Section', () => {
  it('Simple section', () => {
    const input = `Section
section 1
section 1
section 1

section 2

section 3
section 3`
    const html = convert(parse(input), 'projectName')
    expect(pretty(html)).toMatchSnapshot()
  })
})
