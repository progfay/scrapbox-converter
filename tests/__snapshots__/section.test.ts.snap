// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Section Simple section 1`] = `
"<article>
  <h1 class=\\"title\\">Section</h1>
  <section>
    <p>section&nbsp;1</p>
    <p>section&nbsp;1</p>
    <p>section&nbsp;1</p>
  </section>
  <section>
    <p>section&nbsp;2</p>
  </section>
  <section>
    <p>section&nbsp;3</p>
    <p>section&nbsp;3</p>
  </section>
</article>"
`;
